js-exercises/abbreviate-string/abbreviateString.js

```diff
@@ -1,17 +1,19 @@
-function abbreviateString(str) {
-  if (typeof (str) !== 'string') {
-    throw Error(`Expected string, got ${typeof str}`);
+function abbreviateString(word) {
+  const trimWord = word.trim();
```

Comment:
trimWord doesn't sound like an action to you?
An action which needs to be done.
```suggestion
  const trimmedWord= word.trim();
```